templates/named.conf.erb

```diff
@@ -16,5 +16,15 @@ options  {
 include "<%= scope.lookupvar('::dns::localzonepath') %>";
 <% end -%>
 
+<%- unless scope.lookupvar('::dns::acls').empty? -%>
```

Comment:
The reason for this line is to have no acl block at all if the array is empty as to not clutter the config.